ui/src/components/Dropdown/Dropdown.export.js

```diff
@@ -15,183 +15,187 @@ import ListWrapper from './Components/ListWrapper'
-      this.handleOptionListClose()
+        if (event.keyCode === ESCAPE && isDropDownListOpen) {
+            this.handleOptionListClose()
+        }
     }
-  }
-
-  handleItemSelect = (value) => {
-    this.props.onChange(value)
-    this.handleOptionListClose()
-  }
-
-  handleInputKeyDown = (event) => {
-    const SPACEBAR = 32
-    if (event.keyCode === SPACEBAR && !this.state.isDropDownListOpen) {
-      event.preventDefault()
-      this.handleOptionListOpen()
+
+    handleItemSelect = (value) => {
+        this.props.onChange(value)
+        this.handleOptionListClose()
+    }
+
+    handleInputKeyDown = (event) => {
+        const SPACEBAR = 32
+        if (event.keyCode === SPACEBAR && !this.state.isDropDownListOpen) {
+            event.preventDefault()
+            this.handleOptionListOpen()
+        }
+    }
+
+    // RENDERING
+    render() {
+        const {isDropDownListOpen, shownOptions} = this.state
+        const {
+            validationErrorText,
+            enableFilter,
+            placeholder,
+            disabled,
+            height,
+            selectedOption, style:customStyle,
+            width,
+        } = this.props
+
+        const inputComponent = (
+            <ErrorWrapper errorText={validationErrorText}>
+                <Input
+                    disabled={disabled}
+                    isOpen={isDropDownListOpen}
+                    height={height}
+                    onKeyDown={this.handleInputKeyDown}
+                    onClick={this.handleOptionListOpen}
+                    width={width}
+                >
+                    {!selectedOption || !selectedOption.value
+                        ? <Placeholder>{placeholder}</Placeholder>
+                        : (
+                            <SelectedText>
+                                {selectedOption.value}
+                            </SelectedText>
+                        )}
+                </Input>
+            </ErrorWrapper>
+        )
+        const listOptionsComponent = (
+            <ListWrapper>
+                {/* filter */}
+                {enableFilter && (
+                    <Filter
+                        ref={input => {
+                            this.filteringInput = input
+                        }}
+                        value={this.state.filteringInputValue}
+                        onChange={(event) => this.setState({filteringInputValue: event.target.value})}
+                        onKeyPress={this.handleFilteringInputKeyPress}
+                        onKeyDown={this.handleFilteringInputKeyDown}
+                    />
+                )}
+
+                {/* Options */}
+                <ItemsList onClick={this.handleItemSelect} items={shownOptions}/>
+                {shownOptions.length === 0 && <NoMatches/>}
+            </ListWrapper>
+        )
+        return (
+            <div data-disabled={String(disabled)} style={customStyle} className={style.wrapper}>
```

Comment:
where is `customStyle` initialized?

Reply:
it comes from props as style and rename to customStyle